pkg/cephmgr/osd/agent.go

```diff
@@ -396,7 +408,7 @@ func (a *OsdAgent) configureDevices(context *clusterd.Context, devices *DeviceOs
 
 		if scheme.Metadata != nil {
 			// partition the dedicated metadata device
-			if err := partitionBluestoreMetadata(context, scheme.Metadata, context.ConfigDir); err != nil {
+			if err := partitionMetadata(context, scheme.Metadata, context.ConfigDir); err != nil {
 				logger.Errorf("failed to partition bluestore metadata %+v: %+v", scheme.Metadata, err)
```

Comment:
If metadata is applicable to `filestore` now then maybe remove `bluestore` from the log output.